Allwmake

```diff
@@ -32,11 +32,24 @@ log "Building the OpenFOAM-preCICE adapter..."
 # Export the environment variables
 export ADAPTER_PREP_FLAGS
 export ADAPTER_TARGET_DIR
-ADAPTER_PKG_CONFIG_CFLAGS="$(pkg-config --silence-errors --cflags libprecice)"
+
+if pkg-config libprecice; then
+  ADAPTER_PKG_CONFIG_CFLAGS="$(pkg-config --silence-errors --cflags libprecice)"
+  ADAPTER_PKG_CONFIG_LIBS="$(pkg-config --silence-errors --libs libprecice)"
+else
+  log "WARNING: No configuration file 'libprecice.pc' was found in the PKG_CONFIG_PATH."
+  log "Check the preCICE documentation page 'Linking to preCICE' for more details, or proceed if you know what you are doing."
```

Comment:
Do you mean here by `know what you are doing' someone who exports all necessary paths?

Reply:
For example, yes. We want that the "regular" user uses pkg-config, but there are also other such ways one could install preCICE and make it discoverable.

Reply:
Alright. Is 'Linking to precice' the right place to look at/ is there the relevant information? or do we actually want to encourage people to export the `PKG_CONFIG_PATH`? Maybe we should add a few sentences in the adapter wiki and refer to it here?!

Reply:
We currently describe how to export the `PKG_CONFIG_PATH` in the "linking to preCICE" page: https://precice.org/installation-linking.html#make-and-scripts-in-general

I am not sure if we should use URLs in error messages, they tend to get outdated and forgotten.

Reply:
I wouldn't put the link here either. I'm just afraid that the description there is too developer oriented. I think the right place to look at for these users would be the Troubleshooting section on the same page.

Reply:
However, then let's go with this description.